pkg/ranchhand/processhosts.go

```diff
@@ -43,6 +45,13 @@ var (
 			"sudo yum install -y docker-ce-18.06.3.ce-3.el7 containerd.io",
 			"sudo systemctl start docker",
 		},
+		"rhel": {
+			"sudo subscription-manager repos --enable rhel-7-server-extras-rpms || echo 'Error enabling rhel extras repo, continuing...'",
+			"sudo yum install -y yum-utils device-mapper-persistent-data lvm2",
+			"sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo",
+			"sudo yum install -y docker-ce-18.09.2 docker-ce-cli-18.09.2 containerd.io",
+			"sudo systemctl start docker",
```

Comment:
Something that just occurred to me--do these packages enable the docker service by default? I think the RHEL-packaged ones don't, but not sure about these offhand. If they don't, we'd also want to do `sudo systemctl enable --now docker` or something like that.

Reply:
Just looked at one of the provisioned servers, and it is disabled by default:

```[root@rancher-1 ~]# systemctl status docker
● docker.service - Docker Application Container Engine
   Loaded: loaded (/usr/lib/systemd/system/docker.service; disabled; vendor preset: disabled)```

So we should change this, otherwise these nodes won't come up after reboot.

Reply:
👍